### PR TITLE
chore: Enable flutter analyze check

### PR DESCRIPTION
**Pull request by drminh2807**

### CODE REVIEW

File: lib/config/themes.dart [hunk 1]
```diff
@@ -263,83 +251,80 @@ abstract class FluffyThemes {
...
-              } else {
-                return brightness == Brightness.light
+            } else {
+              return brightness == Brightness.light
                   ? LinagoraSysColors.material().outline
                   : LinagoraSysColors.material().outlineDark;
-              }
             }
-          ),
+          }),
```
Comment (hoangdat) [inline lib/config/themes.dart:291]: any wrong here?

File: lib/config/themes.dart [hunk 2]
```diff
@@ -263,83 +251,80 @@ abstract class FluffyThemes {
...
-              else {
-                return brightness == Brightness.light
+            } else {
+              return brightness == Brightness.light
                   ? LinagoraSysColors.material().surface
                   : LinagoraSysColors.material().surfaceDark;
-              }
             }
-          ),
+          }),
```
Comment (hoangdat) [inline lib/config/themes.dart:303]: idem

File: lib/domain/model/room/room_list_extension.dart [hunk 3]
```diff
@@ -4,14 +4,19 @@ import 'package:matrix/matrix.dart';
...
-    int? limit
+    int? limit,
   }) {
-    return where((room) => room.isNotSpaceAndStoryRoom() && room.isShowInChatList())
-      .map((room) => room.toRecentChatSearchModel(matrixLocalizations))
-      .where((model) => model.displayName != null && model.displayName!.contains(keyword))
-      .take(limit ?? length)
-      .toList();
+    return where(
+      (room) => room.isNotSpaceAndStoryRoom() && room.isShowInChatList(),
```
Comment (hoangdat) [inline lib/domain/model/room/room_list_extension.dart:12]: should be a method.

File: lib/domain/usecase/search/pre_search_recent_contacts_interactor.dart [hunk 4]
```diff
@@ -11,17 +11,21 @@ class PreSearchRecentContactsInteractor {
...
 
       for (final room in recentRooms) {
-        final users = room.getParticipants()
-          .where((user) => user.membership.isInvite == true && user.displayName != null)
-          .toSet();
+        final users = room
+            .getParticipants()
+            .where(
+              (user) =>
+                  user.membership.isInvite == true && user.displayName != null,
```
Comment (hoangdat) [inline lib/domain/usecase/search/pre_search_recent_contacts_interactor.dart:22]: should be a method

File: lib/domain/usecase/send_file_interactor.dart [hunk 5]
```diff
@@ -14,12 +14,16 @@ class SendFileInteractor {
...
-        xFile.size,
-        readStream: xFile.readStream,
-      )).toList();
+      final fileInfos = filePickerResult.files
+          .map(
+            (xFile) => FileInfo(
+              xFile.name,
+              xFile.path ?? '${getTemporaryDirectory()}/${xFile.name}',
+              xFile.size,
+              readStream: xFile.readStream,
```
Comment (hoangdat) [inline lib/domain/usecase/send_file_interactor.dart:23]: in the next, we should add a method for transfer from one thing to other thing

File: pubspec.yaml [hunk 6]
```diff
@@ -129,6 +129,8 @@ dependencies:
   open_file: ^3.3.2
   mime: ^1.0.4
   any_link_preview: ^3.0.0
+  async: ^2.11.0
+  cached_network_image: ^3.2.3
```
Comment (hoangdat) [inline pubspec.yaml:133]: what is it?

File: lib/config/themes.dart (hunk 1 above)
Comment (drminh2807) [reply lib/config/themes.dart]: done, seem like dart analyzer do not work in this case

File: pubspec.yaml (hunk 6 above)
Comment (drminh2807) [reply pubspec.yaml:133]: Screenshot 2023-08-09 at 16 36 16
These are fix for that warnings

File: lib/pages/chat/events/video_player.dart [hunk 7]
```diff
@@ -51,7 +51,9 @@ class EventVideoPlayerState extends State<EventVideoPlayer> {
       final networkUri = _networkUri;
       if (kIsWeb && networkUri != null && _chewieManager == null) {
         _chewieManager ??= ChewieController(
-          videoPlayerController: VideoPlayerController.network(networkUri),
+          videoPlayerController: VideoPlayerController.networkUrl(
+            Uri.tryParse(networkUri) ?? Uri(),
```
Comment (hoangdat) [inline lib/pages/chat/events/video_player.dart:55]: tryParse != parse?

**Review by hoangdat — APPROVED** [review]

**Pull request merged**